src/ssl.c

```diff
@@ -55284,6 +55327,85 @@ int wolfSSL_CTX_set_ecdh_auto(WOLFSSL_CTX* ctx, int onoff)
     (void)onoff;
     return WOLFSSL_SUCCESS;
 }
+
+/**
+ * set security level(wolfSSL doesn't suppor security level)
+ * @param ctx  a pointer to WOLFSSL_EVP_PKEY_CTX structure
+ * @param level security level
+ */
+void wolfSSL_CTX_set_security_level(WOLFSSL_CTX* ctx, int level)
+{
+    WOLFSSL_ENTER("wolfSSL_CTX_set_security_level");
+    (void)ctx;
+    (void)level;
+}
+/**
+ * get security level(wolfSSL doesn't suppor security level)
```

Comment:
Add space after "level", "suppor" should be "support"

Reply:
Thanks! Fixed